pom.xml

```diff
@@ -45,6 +45,7 @@
     <module>solrsearch-plugins</module>
     <module>spark-plugins</module>
     <module>transform-plugins</module>
+    <module>format-xls</module>
```

Comment:
modules are sorted alphabetically, move this up to where the other formats are

Reply:
Moved up !